src/dockleHelper.ts

```diff
@@ -134,14 +136,16 @@ export function printFormattedOutput() {
     let titles = [TITLE_VULNERABILITY_ID, TITLE_TITLE, TITLE_SEVERITY, TITLE_DESCRIPTION];
     rows.push(titles);
     dockleOutputJson[KEY_DETAILS].forEach(ele => {
-                let row = [];
-                row.push(ele[KEY_CODE]);
-                row.push(ele[KEY_TITLE]);
-                row.push(ele[KEY_LEVEL]);
-                row.push(ele[KEY_ALERTS][0]);
-                rows.push(row);           
+        if (ele[KEY_LEVEL] != LEVEL_IGNORE && ele[KEY_LEVEL] != LEVEL_INFO) {

```

Comment:
Why skip info?

Reply:
Had a discussion with Vishal on this. He suggested to keep only WARN and FATAL.